TodoListApplication/TodoListSPA/src/app/app-config.ts

```diff
@@ -0,0 +1,85 @@
+import { Configuration } from 'msal';
+import { MsalAngularConfiguration } from '@azure/msal-angular';
+
+// checks if the app is running on IE
+export const isIE = window.navigator.userAgent.indexOf('MSIE ') > -1 || window.navigator.userAgent.indexOf('Trident/') > -1;
+
+/** =================== REGIONS ====================
+ * 
+ * 1) Web API configuration parameters
+ * 2) Authentication configuration parameters
+ * 3) MSAL-Angular specific configuration parameters
+ * 
+ * ================================================= 
+*/ 
+
+
+// #region 1) Web API Configuration
+/** 
+ * Enter here the coordinates of your Web API and scopes for access token request
+ */
+export const apiConfig: {scopes: string[], webApi: string} = {
+    scopes: ['Enter_API_Scopes_Here'],
+    webApi: 'Enter_API_Base_Address_Here',
```

Comment:
The PowerShell automation screws this up. We might need to drop PowerShell support for this sample. Discuss with Tiago once.